.dockerignore

```diff
@@ -1,3 +1,14 @@
 *
```

Comment:
We actually don't need to ignore anything now, right?

Reply:
Not really.

Reply:
I just tried to follow the convention of the current dockerfile. Figured you maybe had reasons for wanting to ignore everything but the necessary files.

Reply:
Can we just remove the `.dockerignore` file then? It'd be weird to have to always update this file every time we add a new top level directory.

Reply:
Will do